### ISSUE TITLE
Always zooms to the top left corner of the current page

### ISSUE THREAD

**Issue by kev23777** [stephanrauh/ngx-extended-pdf-viewer#573]
I looked around at the documentation and didn't see anything. Is there currently any way to allow zooming in on whatever part of the page the user is currently looking at, and if not would that be something that could be added?

**Comment by stephanrauh** [comment]
Do you happen to know if https://mozilla.github.io/pdf.js/web/viewer.html also zooms the top left corner?

I ask because I've added an IE11/Edge patch last week ensuring that the top of the page is visible after paging to the next or to the last page. I don't think that's related, but who know. Better check it twice!

**Comment by kev23777** [comment]
I haven't dug into the code to check, but ng2-pdf-viewer https://github.com/VadimDez/ng2-pdf-viewer is also based on pdfjs and it zooms into where the user is looking. (mostly at least. It does seem to migrate to the right a bit every time you click.)

**Comment by miljann995** [comment]
@stephanrauh I'm facing the same issue. If previous page is visible in the viewer, even little, zooming in or out scrolls to the top left corner of the previous page.
From what I saw in the code, this [line](https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/master/projects/ngx-extended-pdf-viewer/src/assets/viewer-2.6.415.js#L10228) is causing such behaviour. I actually tried calling `this._setScale(val, true)` in `currentScaleValue` setter and that solved the issue. I recorded behaviour for both cases.

This is how it behaves when `this._setScale` is called with `false` for `noScroll` parameter:
![false](https://user-images.githubusercontent.com/16431640/101069579-7dfe8500-359a-11eb-8d98-8e0fe5884176.gif)

And this is how it behaves when `this._setScale` is called with `true` for `noScroll` parameter:
![true](https://user-images.githubusercontent.com/16431640/101069646-940c4580-359a-11eb-85c3-15e112a00b53.gif)
